packages/mdc-top-app-bar/_mixins.scss

```diff
@@ -103,32 +132,3 @@
     }
   }
 }
-
```

Comment:
It was intentional to keep all private mixins at the bottom of the page. Any reason for change of its location?

Reply:
In that case, doesn't it make sense to move `// private` comment lines above `mdc-top-app-bar-mobile-breakpoint_` mixin? This'll avoid touching git history.

Reply:
Yea, that would be better, I'll adjust it.